libraries/lib-music-information-retrieval/tests/TatumQuantizationFitBenchmarkingOutput/summary.txt

```diff
@@ -0,0 +1,5 @@
+AUC: 0.9093477144324603
+B&M Threshold: 0.8423423487195607
+M&S Threshold: 0.9160608375297137
+Octave error RMS: 0.1024786300174427
+Audio file hash: -1840.31005859375
```

Comment:
That's strange -- I thought the checksum would be a summation of samples of audio files, which would tend to average toward 0.